test/e2e/suites/base/schedule_multi_template_test.go

```diff
@@ -0,0 +1,268 @@
+/*
+Copyright 2025 The Karmada Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package base
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"runtime"
+
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/util/rand"
+	"sigs.k8s.io/yaml"
+
+	policyv1alpha1 "github.com/karmada-io/karmada/pkg/apis/policy/v1alpha1"
+	workv1alpha2 "github.com/karmada-io/karmada/pkg/apis/work/v1alpha2"
+	"github.com/karmada-io/karmada/pkg/util/names"
+	"github.com/karmada-io/karmada/test/e2e/framework"
+	testhelper "github.com/karmada-io/karmada/test/helper"
+)
+
+var _ = ginkgo.Describe("[ScheduleMultiTemplate] schedule multi template", func() {
+	var (
+		flinkDeploymentCRD       apiextensionsv1.CustomResourceDefinition
+		flinkDeploymentNamespace string
+		flinkDeploymentName      string
+		flinkDeploymentObj       *unstructured.Unstructured
+		flinkDeploymentGVR       schema.GroupVersionResource
+	)
+
+	ginkgo.BeforeEach(func() {
+		ginkgo.By("load FlinkDeployment CRD from manifest file", func() {
+			// Get the directory where this test file is located
+			_, filename, _, _ := runtime.Caller(0)
+			testDir := filepath.Dir(filename)
+			// This file is download from https://raw.githubusercontent.com/apache/flink-kubernetes-operator/release-1.13/helm/flink-kubernetes-operator/crds/flinkdeployments.flink.apache.org-v1.yml"
+			crdPath := filepath.Join(testDir, "manifest", "flinkdeployments.flink.apache.org-v1.yaml")
+
+			crdYAML, err := os.ReadFile(crdPath)
+			gomega.Expect(err).ShouldNot(gomega.HaveOccurred(), "Failed to read FlinkDeployment CRD from %s", crdPath)
+
+			err = yaml.Unmarshal(crdYAML, &flinkDeploymentCRD)
```

Comment:
```suggestion
			// Get the directory where this test file is located
			_, filename, _, _ := runtime.Caller(0)
			testDir := filepath.Dir(filename)
			// This file is download from https://raw.githubusercontent.com/apache/flink-kubernetes-operator/release-1.13/helm/flink-kubernetes-operator/crds/flinkdeployments.flink.apache.org-v1.yml"
			crdPath := filepath.Join(testDir, "manifest", "flinkdeployments.flink.apache.org-v1.yaml")

			crdYAML, err := os.ReadFile(crdPath)
			gomega.Expect(err).ShouldNot(gomega.HaveOccurred(), "Failed to read FlinkDeployment CRD from %s", crdPath)

			err = yaml.Unmarshal(flinkDeploymentCRDConfig, &flinkDeploymentCRD)
```
This is an example of how `//go:embed` simplifies the code. 
But with this we might don't need to the whole `load FlinkDeployment CRD from manifest file` at all.